k8s/controllers/solution/instance_controller_test.go

```diff
@@ -257,26 +257,26 @@ var _ = Describe("Instance controller", Ordered, func() {
 		})
 	})
 
-	Describe("Solution Events", func() {
-		When("the solution referenced by the instance is changed", func() {
-			var requests []ctrl.Request
-			BeforeEach(func(ctx context.Context) {
-				By("simulating a call to the handleSolution function")
-				requests = controller.handleSolution(solution)
-			})
-
-			It("should return a request for the instance", func() {
-				Expect(requests).To(ContainElement(ctrl.Request{NamespacedName: DefaultInstanceNamespacedName}))
-			})
-		})
-	})
+	// Describe("Solution Events", func() {
```

Comment:
We do we remove this?

Reply:
This test is broken because of upgrading controller runtime or env test. It looks like a test issue that the instance_webhook is not registered automatically in the suite test. 
I have changed the 01.update to cover that part of logic - when solution or target changes, instance will be reconciled and deployment will take the change.

Reply:
Jiawei helped to fix the test issue by adding index to fakeclient manually. The UT is enabled again now.